f8a_report/report_helper.py

```diff
@@ -743,9 +742,20 @@ def get_report(self, start_date, end_date, frequency='daily'):
 
         if len(ids) > 0:
             worker_result = self.retrieve_worker_results(
-                start_date, end_date, ids, ['stack_aggregator_v2'], frequency)
+                start_date, end_date, ids, ['stack_aggregator_v2'], frequency, retrain)
             return worker_result, ingestion_results
         else:
             logger.error('No stack analyses found from {s} to {e} to generate an aggregated report'
                          .format(s=start_date, e=end_date))
             return False, ingestion_results
+
+    def retrain(self, start_date, end_date, frequency='weekly', retrain='T'):
```

Comment:
Please add unit test cases for this new function

Reply:
yep adding